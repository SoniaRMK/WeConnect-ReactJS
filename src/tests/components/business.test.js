import {shallow} from 'enzyme';
import React from 'react';
import BusinessOne from '../../components/business/business';

describe('BusinessOne component',() => {

    it('should match snapshot',() =>{
        const wrapper = shallow(<BusinessOne/>)
        expect(wrapper).toMatchSnapshot();
    })
})
