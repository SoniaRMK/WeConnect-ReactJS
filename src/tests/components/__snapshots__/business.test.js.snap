// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BusinessOne component should match snapshot 1`] = `
<Route
  render={[Function]}
/>
`;
